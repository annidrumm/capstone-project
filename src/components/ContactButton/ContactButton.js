import React from 'react'
import styled from 'styled-components/macro'

export default function ContactButton() {
  return (
    <AnchorStyled href="mailto:influencer@gmail.com">
      <ContactButtonStyled>Contact</ContactButtonStyled>
    </AnchorStyled>
  )
}

const ContactButtonStyled = styled.button`
  color: var(--font-color-white);
  font-family: var(--font-weight-light);
  font-size: 20px;
  background: var(--darkgreen);
  border: var(--darkgreen);
  border-radius: 35px;
  box-shadow: 0 3px 10px #0000008b;
  width: 140px;
  height: 75px;
  text-align: center;
  display: block;
  margin: auto;
  border: none;
  margin-bottom: 40px;
  cursor: pointer;
`
const AnchorStyled = styled.a`
  text-decoration: none;
`
