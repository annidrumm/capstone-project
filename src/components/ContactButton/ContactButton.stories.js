import React from 'react'
import ContactButton from './ContactButton'

export default {
  title: 'Components/Button',
  component: ContactButton,
  argTypes: {
    headline: { control: 'text' },
  },
}

const Template = (args) => <ContactButton {...args} />

export const contact = Template.bind({})
contact.args = {}
