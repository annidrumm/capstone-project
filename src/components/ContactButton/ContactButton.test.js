import React from 'react'
import renderer from 'react-test-renderer'
import ContactButton from './ContactButton'
import '@testing-library/jest-dom/extend-expect'

describe('ContactButton', () => {
  it('renders ContactButton correctly', () => {
    const tree = renderer.create(<ContactButton />)
    expect(tree).toMatchSnapshot()
  })
})
