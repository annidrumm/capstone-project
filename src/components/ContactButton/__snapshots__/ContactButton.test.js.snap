// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactButton renders ContactButton correctly 1`] = `
<a
  href="mailto:influencer@gmail.com"
  style={
    Object {
      "textDecoration": "none",
    }
  }
>
  <button
    className="ContactButton__ContactButtonStyled-sc-1kv6ekb-0 beJosE"
  >
    Contact
  </button>
</a>
`;
